package.json

```diff
@@ -11,7 +11,9 @@
     "react": "^17.0.2",
     "react-color-extractor": "^1.1.2",
     "react-dom": "^17.0.2",
+    "react-elastic-carousel": "^0.11.5",
     "react-icons": "^4.3.1",
+    "react-multi-carousel": "^2.8.2",
```

Comment:
Is this package used anywhere?

Reply:
No, you can remove _**react-multi-carousel**_ by npm uninstall react-multi-carousel. I tried it but removed it later on.